src/cli/cli.cc

```diff
@@ -1771,23 +1763,15 @@ int main (const int argc, const char* argv[]) {
         buildArgs << " --test=true";
       }
 
-      auto process = new SSC::Process(
-        settings["build_script"],
-        buildArgs.str(),
-        fs::current_path().string(),
-        [](SSC::String const &out) { stdWrite(out, false); },
-        [](SSC::String const &out) { stdWrite(out, true); },
-        [](SSC::String const &code) {
-          if (std::stoi(code) != 0) {
-            log("build failed, exiting with code " + code);
-            // TODO(trevnorris): Force non-windows to exit the process.
-            exit(std::stoi(code));
-          }
-        }
-      );
-
-      process->open();
-      process->wait();
+      // Removed new SSC::Process() style code, caused random crashes
```

Comment:
cc @heapwolf @chicoxyzzy @trevnorris

Reply:
IIRC the process wrapper was necessary to get the stdout and stderr

Reply:
@chicoxyzzy stderr is being redirected to stdout now:
https://github.com/socketsupply/socket/blob/create-socket-win32-setup/src/process/process.hh#L51
```
inline ExecOutput exec (SSC::String command) {
    command = command + " 2>&1";
```

stdout goes to ExecOutput.output and will be written to ssc stdout if error code != 0.

Reply:
ExecOutput provides streamed (not blocking) output, and exit code handling consistently across all platforms, exec will block and not show output to the user so they won't know if their script is making progress.

Reply:
we should just figure out the root problem, what caused the random crashes, because removing this code will break a lot of other use cases.